### ISSUE TITLE
Is possible to change tile height?

### ISSUE THREAD

**Issue by Marabita** [prolificinteractive/material-calendarview#87]
I'm trying to change the height of the tiles.
I adjusted the tiles to width of screen width divided by 7

```java
        Display display = getWindowManager().getDefaultDisplay();
        Point size = new Point();
        display.getSize(size);
        int width = size.x;

        mCalendarView.setTileSize(width / 7);  // will make calendar fit on width screen
```

The height follows the width size, and the calendar stay too big (see image)
![device-2015-07-19-153810](https://cloud.githubusercontent.com/assets/1102623/8767239/48258e2c-2e2c-11e5-86bb-a9d5b2eeb155.png)

if you can change the height could have something like this:
![device-2015-07-19-153640](https://cloud.githubusercontent.com/assets/1102623/8767249/a46c5ee0-2e2c-11e5-9c7e-351ae05367e5.png)

**Comment by dandc87** [comment]
TL;DR, It's not possible, and will likely stay that way.

A major issue I have with allowing width and height to be independent is, how would the selection indicator work? Should it limited to the day's cell, or allowed to be drawn outside the box? If one was chosen, there would be people who want the other, or for it to be customizable per day. It's just a black hole of complexity. A line had to be drawn somewhere, and this library currently draws it at each day is a square. It makes everything fairly simple, and renders a visually pleasing calendar.

**Comment by dandc87** [comment]
In version 0.8.0 there is a new sizing algorithm which can automatically size to fit widths. The issue of showing a blank bottom row will always exist while we still have horizontal swiping, so I'm going to close this. Though I would suggest enabling `showOtherDates` so its not blank.

**Comment by Marabita** [comment]
Great @dandc87 !!

**Comment by Marabita** [comment]
Hi, im using new version and get some points:
mCalendarView.setTopbarVisible(false);  maybe just turning view to invisible not to gone

![2015-08-27 16 09 20](https://cloud.githubusercontent.com/assets/1102623/9526010/53f8735e-4cbd-11e5-854e-fd20b45e8f45.png)

**Comment by dandc87** [comment]
@Marabita there should be a fix in 0.8.1
